pygmt/helpers/tempfile.py

```diff
@@ -154,14 +154,12 @@ def tempfile_from_geojson(geojson):
             # Other 'geo' formats which implement __geo_interface__
```

Comment:
The pyogrio package doesn't support the `schema` parameter. With geopandas 1.0 released, pyogrio will be the default engine, and then the code at lines 136-150 will break.

We need to find a way to support both geopandas 0.x and 1.x.